### DISCUSSION TITLE
能否支持文章自动生成目录呢？

### DISCUSSION THREAD

**Discussion by FunkyGod** [halo-dev/halo#5406] [Ideas]
### 你当前使用的版本

2.12.0

### 描述一下此特性

默认文章没有目录，能否生成悬浮目录呢

**Comment by ruibaby** [comment]
这个特性看起来写一个通用的插件可能会更加合适，目前已经有了[内容的扩展点](https://docs.halo.run/developer-guide/plugin/api-reference/server/extension-points/post-content)。

/area plugin

**Comment by ruibaby** [comment]
我将把此 issue 转移到 Discussion 的 Ideas 分类中，看看社区是否有人愿意实现此插件。

**Reply by FunkyGod** [reply]
求大佬实现，感谢
